src/glog/logging.h.in

```diff
@@ -448,7 +468,12 @@ DECLARE_int32(minloglevel);
 
 // If specified, logfiles are written into this directory instead of the
 // default logging directory.
+
+#if defined(GLOG_OS_WINDOWS) && defined(UNICODE)
+DECLARE_wstring(log_dir);
```

Comment:
Changing from `std::string` to `std::wstring` in case `UNICODE` is defined will break user code.

There are two options here:
1. Make `log_dir` accept both `std::string` and `std::wstring` by introducing a proxy type that performs the necessary conversions.
2. (Unconditionally) provide `log_wdir` defined in terms of `std::wstring`.

Reply:
Yeah，you're right, I didn't think enough about it.